app/Http/Controllers/ProductsController.php

```diff
@@ -1,29 +1,30 @@
 <?php
 
 namespace App\Http\Controllers;
-
 use App\Product;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\File;
 
 class ProductsController extends Controller
+
 {
     /**
      * Display a listing of the resource.
      *
      * @return \Illuminate\Http\Response
      */
     public function index()
+
     {
```

Comment:
Opening brace should be on the line after the declaration; found 1 blank line(s)